src/app/containers/StoryPromo/MediaIndicator/__snapshots__/index.test.jsx.snap

```diff
@@ -11,6 +11,17 @@ exports[`Story Promo Media Indicator should not render if item media object has
 />
 `;
 
+exports[`Story Promo Media Indicator should render Index Also video indicator correctly 1`] = `
```

Comment:
Is this snap reflective of what we want? Perhaps we need a deep snapshot?

Reply:
I replaced it with `shouldMatchSnapshot` and added an assertion test. Let me know if you think that with that is enough :)